app/src/main/res/drawable/background_gradient_rainy.xml

```diff
@@ -1,9 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <shape xmlns:android="http://schemas.android.com/apk/res/android" android:shape="rectangle" >
   <gradient
-      android:angle="90"
-      android:startColor="#C8E4F8"
-      android:centerColor="#C8E4F8"
-      android:endColor="#7CD9F5"
-      android:type="linear" />
+      android:gradientRadius="140%"
```

Comment:
why is it 140% and not 100%? See this:
https://stackoverflow.com/questions/11176590/gradient-radius-as-percentage-of-screen-size